packages/clerk-js/src/core/state.ts

```
@@ -0,0 +1,33 @@
+import type { State as StateInterface } from '@clerk/types';
+import { computed, effect } from 'alien-signals';
+
+import { eventBus } from './events';
+import { signInComputedSignal, signInErrorSignal, signInSignal } from './signals';
+import { BaseResource } from './resources/Base';
+import { SignIn } from './resources/SignIn';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)